scripts/RepoLayout.props

```diff
@@ -9,7 +9,7 @@
     <!-- Set version for runtimeconfig.json, used by DotnetRuntimeBootstrapper for minimal version to support
          To be kept in sync with the .NET SDK version in global.json
     -->
-    <RuntimeFrameworkVersion>6.0.9</RuntimeFrameworkVersion>
+    <RuntimeFrameworkVersion>6.0.10</RuntimeFrameworkVersion>

```

Comment:
We shouldn't be changing this, as it completely messes the developer experience and requires a specific version installed locally. For example, I now have installed 6.0.403, and this settings forces me to have 6.0.402 installed.
This should stay at `6.0.0`, and only get updated on AppVeyor:
https://github.com/gitextensions/gitextensions/blob/4ee6417f4a7bb00e30e13ccef639f5e247b3aae0/appveyor.yml#L21-L22

Reply:
Update in all local builds, so we use the expected in test builds?

Reply:
I am tired, and my brain is fried... and I don't understand the question, sorry.

Locally, we build with whatever SDK we have installed - 6.0.400 or 6.0.456. The runtime version depends on the installed SDK - 6.0.8 or 6.0.11...   Forcing the devs to install 6.0.10 is unnecessary.
When we do CI builds though, we will build against the latest available, and this runtime will be required for users to run the app.

Reply:
I use the Debug builds in my normal usage. Then it would be good if the same runtime is used as a release build would require.
Could be a separate argument too.

Reply:
@gerhardol  if you want to run latest all you need to do is run the .\Scripts\Update-DotnetVersion.ps1 script just as appveyor does.  Then build as normal.  
![image](https://user-images.githubusercontent.com/3629489/201813996-58005c0d-eea4-4532-a3f1-4cc04dac0402.png)

![image](https://user-images.githubusercontent.com/3629489/201814190-b5a632a6-a5dc-49a0-883f-21cac2a3d558.png)

![image](https://user-images.githubusercontent.com/3629489/201814122-f57790be-a2e7-46a7-af7a-d70ef8483638.png)
![image](https://user-images.githubusercontent.com/3629489/201814275-83c933bb-22af-48f3-9be9-0ef324633678.png)

You can even have a build script that is ignored that does this for you.

```ps

$path = Split-Path -Path $PSCommandPath -Parent
$update =Join-Path -Path $path -ChildPath Update-DotnetVersion.ps1

& $update

dotnet build --no-incremental --use-current-runtime

```
Something like 
![image](https://user-images.githubusercontent.com/3629489/201816169-48e72da7-7e65-4208-b3f8-ddf4477b485c.png)

You can play with --use-current-runtime or not.  But  using a git ignore 
![image](https://user-images.githubusercontent.com/3629489/201816679-9076727e-f5de-4d0b-baf0-ad699eac1b5e.png)
You can then build your project with just a powershell script run. See scripts\Bisect-Test.ps1 for ideas on building and then running.

Reply:
You could even auto install the sdk and desktop with chocolatey.

```ps
choco upgrade dotnet-6.0-sdk -y
choco upgrade dotnet-6.0-desktopruntime -y
```

Reply:
'Can' is not the same as using the same setup as users by default.
I would like me and other pre-release users to have the same environment as users will have.
In a build and in a debugger is a slightly different situation.

Reply:
If a build comes from Appveyor than it'd target the latest available runtime. If a developer builds locally the build can target the installed runtime. There's generally no reason to force a developer to install the latest or specific version of runtime. And, if there's a compelling reason, e.g. a fixed bug or security vulnerability, then we'll bump the minimum required version in global.json.